tools/cli/tests/start-demo.test.js

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const execSync = require('child_process').execSync;
+const { exec } = require('child_process');
+const path = require('path');
+const rimraf = require('rimraf');
+
+const componentFolder = 'create-component-test-folder';
+const componentName = 'test-name';
+const componentFolderPath = path.resolve(path.join(process.cwd(), componentFolder));
+const componentSourcePath = path.join(componentFolderPath, componentName);
+
+describe('Start demo test', () => {
+    afterAll(() => {
+        rimraf.sync(componentFolderPath);
+    });
+
+    test('Start a dev server hosting the demo', (done) => {
+        // create a component
+        execSync(`node index.js create ${componentName} ./${componentFolder}`, { encoding: 'utf8' });

```

Comment:
Does it makes sense to inherit the stdio of all the execSync process so their output is displayed in the terminal?
`execSync(..., {stdio:'inherit'})`